packages/babel-make-styles/__fixtures__/shared-mixins/code.ts

```diff
@@ -0,0 +1,7 @@
+import { makeStyles } from '@fluentui/react-make-styles';
+import { sharedStyles } from './mixins';
+
+export const useStyles = makeStyles({
```

Comment:
Is is also possible to have 'smarter' mixins ? 

```typesript
const useStyles = makeStyles(smartMixin(option1, option2....)
```

Reply:
I can imagine someone wanting to do this because it's 'better'

Reply:
Nope, it will fail currently. I would postpone this until a real ask 🐱